### ISSUE TITLE
crossOrigin attribute should be lowercased in rendered

### ISSUE THREAD

**Issue by upsuper** [facebook/react#14829]
**Do you want to request a *feature* or report a *bug*?**

A bug.

**What is the current behavior?**

When using
```html
<img crossOrigin="anonymous"/>
```
it is rendered as
```html
<img crossOrigin="anonymous">
```

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code and it doesn't have dependencies other than React. Paste the link to your JSFiddle (https://jsfiddle.net/Luktwrdm/) or CodeSandbox (https://codesandbox.io/s/new) example below:**

https://jsfiddle.net/upsuper/y9q5fp3t/

**What is the expected behavior?**

`crossOrigin` should be lower-cased when rendering.

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**

Latest version.

(This is basically just #14216, but that was closed because HTML attribute name is case-insensitive. However, this is still a problem for SVG, since SVG attribute name is case-sensitive, and thus `crossOrigin` doesn't work.)

**Comment by aweary** [comment]
Thanks for bringing this up, we already special case `tabIndex` for SVG so we should do the same here.

**Comment by gaearon** [comment]
Fixed in 16.8.2.
